cmd/tegola/cmd/cache.go

```diff
@@ -279,35 +170,304 @@ var cacheCmd = &cobra.Command{
 
 type MapTile struct {
 	MapName string
-	Tile    *tegola.Tile
+	Tile    *slippy.Tile
+}
+
+type Format struct {
+	X, Y, Z uint
+	Sep     string
+}
+
+var ErrFormat = errors.New("Invalid format")
+// z/x/y
+var df = Format{
+	X:   1,
+	Y:   2,
+	Z:   0,
+	Sep: "/",
+}
+
+func NewFormat(format string) (Format, error) {
```

Comment:
Can we get a test for this function demonstrating single digit and multi digit processing?

Reply:
Scrap the "multi digit process" part of this test request. I misinterpreted what this func was doing. Still could use a test though.

Reply:
I'll add tests